.travis.yml

```diff
@@ -38,3 +38,6 @@ script:
         source ./.CI/build_all;
       fi
 
+after_failure:
+  - "ls -l /Users/travis/miniconda/conda-bld/work/pybufr-ecmwf-0.82/build/lib.macosx-10.6-x86_64-2.7/pybufr_ecmwf/" 
+  - "cat /Users/travis/miniconda/conda-bld/work/pybufr-ecmwf-0.82/build/lib.macosx-10.6-x86_64-2.7/pybufr_ecmwf/f2py_build.log"
```

Comment:
thanks.
I have seen this problem with undefined symbols before on MacOSX machines. My understanding is that it has to do with mixing objects produced by different compiler versions (clang vs gcc?). The way I solved it was to use homebrew to install fresh gcc and gfortran copies AND at the same time a new python runtime that was compiled by this same gcc. This at least is how I managed to get the travis-ci tests working for MacOSX.
Since MacOSX provides clang, even if you ask for gcc, this may easily be overlooked.
Also, since gfortran usually is build with gcc other options will not work, unless you can get your hands on (1) a gfortran version compiled using clang, and a python runtime version compiled with the same version of clang.